src/python/doc/installation.rst

```diff
@@ -174,9 +175,8 @@ Documentation
 
 To build the documentation, `sphinx-doc <http://www.sphinx-doc.org>`_ and
 `sphinxcontrib-bibtex <https://sphinxcontrib-bibtex.readthedocs.io>`_ are
-required. As the documentation is auto-tested, `CGAL`_, `Eigen`_,
-`Matplotlib`_, `NumPy`_, `POT`_, `Scikit-learn`_ and `SciPy`_ are
-also mandatory to build the documentation.
+required. As the documentation is auto-tested, all the :ref:`Optional third-party libraries`
+are also mandatory to build the documentation.
```

Comment:
Er, no, not all. You don't need torch/eagerpy/hnswlib/pykeops to build the doc.

Reply:
Sorry about that. I thought the list had grown but didn't check. I rolled back on 260570c